src/xdocs/checks/blocks/rightcurly.xml

```diff
@@ -172,33 +181,40 @@ public class Test {
         </source>
         <p id="Example2-code">Example:</p>
         <source>
-public class Test {
+public class Example2 {
 
   public void test() {
 
+    boolean foo = false;
     if (foo) {
       bar();
-    } else { bar(); }   // violation, right curly must be alone on line
+    } else { bar(); }
+    // violation above, ''}' at column 21 should be alone on a line.'
 
     if (foo) {
-      bar();
+        bar();
```

Comment:
Please keep indentation as it was before, as +2.

Please apply to all updates like this

Reply:
done @romani